Description: Upstream was using the wrong package name.
 Upstream was using i instead of l, this fixes it.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2015-03-23

--- horizon-2015.1~b3.orig/horizon/templates/horizon/_scripts.html
+++ horizon-2015.1~b3/horizon/templates/horizon/_scripts.html
@@ -11,7 +11,7 @@
 {% compress js %}
 <script type="text/javascript">var STATIC_URL = "{{ STATIC_URL }}";</script>
 <script src='{{ STATIC_URL }}horizon/lib/angular/angular-cookies.js'></script>
-<script src="{{ STATIC_URL }}horizon/lib/angular/irdragndrop.js"></script>
+<script src="{{ STATIC_URL }}horizon/lib/angular/lrdragndrop.js"></script>
 <script src="{{ STATIC_URL }}horizon/lib/angular/smart-table.js"></script>
 
 <script src='{{ STATIC_URL }}horizon/js/angular/controllers/namespace-controller.js'></script>
--- horizon-2015.1~b3.orig/horizon/templates/horizon/jasmine/jasmine.html
+++ horizon-2015.1~b3/horizon/templates/horizon/jasmine/jasmine.html
@@ -15,7 +15,7 @@
   <script src="{{ STATIC_URL }}horizon/lib/angular/angular-cookies.js"></script>
   <script src="{{ STATIC_URL }}horizon/lib/angular/angular-bootstrap.js"></script>
   <script src="{{ STATIC_URL }}horizon/lib/angular/smart-table.js"></script>
-  <script src="{{ STATIC_URL }}horizon/lib/angular/irdragndrop.js"></script>
+  <script src="{{ STATIC_URL }}horizon/lib/angular/lrdragndrop.js"></script>
   <script src="{{ STATIC_URL }}horizon/lib/d3.js"></script>
 
   <script type="text/javascript">
--- horizon-2015.1~b3.orig/openstack_dashboard/static_settings.py
+++ horizon-2015.1~b3/openstack_dashboard/static_settings.py
@@ -20,7 +20,7 @@ to match their distribution's standards.
 import xstatic.main
 import xstatic.pkg.angular
 import xstatic.pkg.angular_bootstrap
-import xstatic.pkg.angular_irdragndrop
+import xstatic.pkg.angular_lrdragndrop
 import xstatic.pkg.angular_smart_table
 import xstatic.pkg.bootstrap_datepicker
 import xstatic.pkg.bootstrap_scss
@@ -46,7 +46,7 @@ STATICFILES_DIRS = [
     ('horizon/lib/angular',
         xstatic.main.XStatic(xstatic.pkg.angular_bootstrap).base_dir),
     ('horizon/lib/angular',
-        xstatic.main.XStatic(xstatic.pkg.angular_irdragndrop).base_dir),
+        xstatic.main.XStatic(xstatic.pkg.angular_lrdragndrop).base_dir),
     ('horizon/lib/angular',
         xstatic.main.XStatic(xstatic.pkg.angular_smart_table).base_dir),
     ('horizon/lib/bootstrap_datepicker',
--- horizon-2015.1~b3.orig/requirements.txt
+++ horizon-2015.1~b3/requirements.txt
@@ -42,7 +42,7 @@ six>=1.9.0
 XStatic>=1.0.0  # MIT License
 XStatic-Angular>=1.3.7  # MIT License
 XStatic-Angular-Bootstrap>=0.11.0.2  # MIT License
-XStatic-Angular-Irdragndrop>=1.0.2.1  # MIT License
+XStatic-Angular-Lrdragndrop>=1.0.2.1  # MIT License
 XStatic-Bootstrap-Datepicker>=1.3.1.0  # Apache 2.0 License
 XStatic-Bootstrap-SCSS>=3  # Apache 2.0 License
 XStatic-D3>=3.1.6.2  # BSD License (3 clause)
